backend/molgenis-emx2/src/main/java/org/molgenis/emx2/utils/PeriodConverter.java

```diff
@@ -0,0 +1,37 @@
+package org.molgenis.emx2.utils;
```

Comment:
please put this file in the molgenis-emx2-sql module, unless impossible.
because that is the place where jooq is used.

Reply:
Not possible because of circular dependency